### PR TITLE
Math Magicians: Components - Day 1

### PR DESCRIPTION
**Pull request by hajnaloltyan**
## Math Magicians: Components - Day 1

In this PR I:
* Deleted all the boilerplate files
* Created Components folder with Calculator files
* Styled the Calculator based on the given design
* Created child component Buttons with styles
* [Live Demo](https://hajnaloltyan.github.io/react-calculator/)

### CODE REVIEW

**Review by GzimAsani — CHANGES_REQUESTED** [review]
Hi @hajnaloltyan , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have anything under future features on your readme file. According to the requirements you need to add 1-3 cases. Please do that.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by GzimAsani — CHANGES_REQUESTED** [review]
Hi @hajnaloltyan , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have anything under future features on your readme file. According to the requirements you need to add 1-3 cases. Please add them under here: 
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Calculator/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './Calculator.css';
+
+const Calculator = () => (
```
Comment (1GORDON) [inline src/components/Calculator/Calculator.js:4]: Great job @hajnaloltyan 🤝

- [ ] As stated in the project requirements, kindly ensure the Calculator.js should have at least 1 child component.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @hajnaloltyan ✋

Good job so far!

## Highlights!
- Calculator.js is a functional component ✔️ 
- All create-react-app boilerplate were removed ✔️
- App.js imports Calculator.js ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @hajnaloltyan 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Highlights
- You implemented all suggested changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**